examples/orion_example.py

```diff
@@ -14,183 +18,117 @@
 logger = logging.getLogger(__name__)
```

Comment:
please configure basicConfig here for logging. Otherwise the clients will only log with error level